mautic/mautic
# The E-Mail Preview Url is set to published when the email is published

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The E-Mail Preview Url is set to published when the email is published.

![image](https://github.com/mautic/mautic/assets/96085911/0eb70f42-dc22-4434-8961-8ae0ba9c4164)
![image](https://github.com/mautic/mautic/assets/96085911/e87d455a-df2b-4036-ae24-5762d1ed94eb)



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Toggle the email publish status in the email list and email view.
3. E-Mail Preview Url status also changes. It had not changed before.
